addons/medical/functions/fnc_isMedic.sqf

```diff
@@ -19,7 +19,7 @@
 
 params ["_unit", ["_medicN", 1]];
 
-private _class = _unit getVariable [QGVAR(medicClass), getNumber (configFile >> "CfgVehicles" >> typeOf _unit >> "attendant")];
+private _class = _unit getVariable [QGVAR(medicClass), _unit getUnitTrait "medic"];
```

Comment:
ACE_common_fnc_getUnitTrait that will check all traits? Input of what you need to check. This will also help later if we decide to implement more types of medics, engineers and more.
Thoughts?